ci_scripts/check_documentation.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+log_file="${TMPDIR}/jazzy_status.log"
+
+if ! command -v jazzy > /dev/null; then
+  if [[ "${CI}" != "true" ]]; then
+    echo "ERROR: Please install jazzy:"
+    echo "https://github.com/realm/jazzy"
+    exit 1
+  fi
+
+  echo "Installing jazzy..."
+
+  gem install jazzy || die "Executing \`gem install jazzy\` failed"
```

Comment:
Why `info` and `die` and not what I have? I copied this from the other Travis ci scripts like `check_fauxpas.sh` and `run_analyzer.sh`  (whereas export build doesn't actually need to trigger Travis failures so I wasn't sure it was really the same sort of thing).